### DISCUSSION TITLE
一个工单有4个sql，这4个sql执行顺序

### DISCUSSION THREAD

**Discussion by hubuyt001** [hhyo/Archery#2466] [General]
一个工单有4个sql，这4个sql执行顺序是不是1，2，3，4 按照顺序单线程执行

**Comment by LeoQuote** [comment]
你实验一下不就知道了，四条插入语句，值是当前时间，期待你的测试结果

**Comment by hubuyt001** [comment]
是顺序执行的

Leo Q ***@***.***>于2023年12月22日 周五23:41写道：
